src/aria2p/interface.py

```diff
@@ -577,6 +629,65 @@ def process_keyboard_event_main(self, event):
         elif event.key_code in Keys.UN_SELECT_ALL:
             pass  # TODO
 
+        elif event.key_code in Keys.MOVE_HOME:
+            if self.focused > 0:
+                self.focused = self.rows.index(self.rows[0])
```

Comment:
Based on the description you give ("move to first download"), I understand it moves focus to the absolute top of the window, and not the currently displayed top row? Then you can simply do:

```suggestion
                self.focused = 0
```